Code_Exercises/Functors/README.md

```diff
@@ -0,0 +1,40 @@
+# SYCL Academy
+
+## Exercise 20: Functors
```

Comment:
We no longer number the exercises

Reply:
Resolving: The individual readmes are still numbered to match the main table